code/controllers/subsystems/discord.dm

```diff
@@ -47,11 +48,49 @@ SUBSYSTEM_DEF(discord)
 /datum/controller/subsystem/discord/Initialize()
 	GLOB.config.load("config/discord.txt", "discord")
 	update_channels()
+	initialize_webhooks()
 
 /datum/controller/subsystem/discord/stat_entry(msg)
-	msg = "A: [active] C: [length(channels)] G: [length(channels_to_group)]"
+	msg = "A: [active] | C: [length(channels)] | G: [length(channels_to_group)] | W: [length(webhooks)]"
 	return ..()
 
+/*
+ * Proc initialize_webhooks
+ * Used to load the webhooks from the db or the flatfile
+ * Wipes all current channels and channel maps.
+ *
+ * @return num	- Error code. 0 upon success.
```

Comment:
```suggestion
 * Return FALSE on success, TRUE otherwise
```
Why are we returning FALSE on success anyways? If this runtimes and returns null, you'd not catch it with the (easy) logic on whatever uses this